client/components/info-popover/index.jsx

```diff
@@ -68,30 +87,39 @@ export default class InfoPopover extends Component {
 
 	render() {
 		return (
-			<span
-				onClick={ this.handleClick }
-				ref="infoPopover"
-				className={ classNames(
-					'info-popover',
-					{ is_active: this.state.showPopover },
-					this.props.className
-				) }
-			>
-				<Gridicon icon={ this.props.icon } size={ this.props.iconSize } />
-				<Popover
-					autoRtl={ this.props.autoRtl }
-					id={ this.props.id }
-					isVisible={ this.state.showPopover }
-					context={ this.refs && this.refs.infoPopover }
-					ignoreContext={ this.props.ignoreContext }
-					position={ this.props.position }
-					onClose={ this.handleClose }
-					className={ classNames( 'popover', 'info-popover__tooltip', this.props.className ) }
-					rootClassName={ this.props.rootClassName }
+			<Fragment>
+				<span
+					role="button"
```

Comment:
❓ I guess we could also use a `button` element instead of assigning `role=button` to `span`? I'm not an accessibility expert, just curious - not crucial for this PR.